chapter2/tests/t11/pool.d/slowww.conf

```diff
@@ -0,0 +1,466 @@
+; Start a new pool named 'www'.
+; the variable $pool can be used in any directive and will be replaced by the
+; pool name ('www' here)
+[slowww]
+
+; Also added for the correct execution of the slow report
+php_admin_value[max_execution_time] = 605
+
+; Per pool prefix
```

Comment:
all default values should be defined globally in php-fpm.conf and `concrete` pools should only define the "DIFF" that way you can clearly see what's happening in each pool. Right now it's hard.